### PR TITLE
feat: Missing payment and main menses tag cases added to incomplete page

### PR DESCRIPTION
**Pull request by SaraVieira**
This PR will add the following

- If you have not payed you will be redirect to incomplete instead of profile
- If you did not select a mensies tag one will be asked

### CODE REVIEW

File: src/features/payment/PaymentForm.js [hunk 1]
```diff
@@ -206,11 +206,6 @@ const PaymentForm = ({ standalone = true, submitLabel, onDone = () => {} }) => {
   } else {
     return (
       <>
-        {paymentStatus === "unfinished" && (
```
Comment (SaraVieira) [inline src/features/payment/PaymentForm.js:209]: I removed this since it's already taken care of by the text at the top, let me know your thoughts

**Comment by SaraVieira** [comment]
CleanShot 2021-05-18 at 19 50 23

**Review by raae — COMMENTED** [review]
**thought:** Would it be nicer for the user to get a complete overview of what is incomplete? 

It could look similar to the profile page? One card for the "settings" and one for payment? Not sure about this though.

**Comment by SaraVieira** [comment]
> Would it be nicer for the user to get a complete overview of what is incomplete?

Main reason I did not do it is because of being such an edge case but if it does happen then yeah it would be nicer but on the other hand when you pay you have to leave the page and come back so I'm not sure

**Comment by raae** [comment]
Because of the needs of Netlify functions we are using npm, so remove yarn.lock and use package.lock from main. Otherwise I think Netlify gets confused.

File: src/pages/incomplete.js [hunk 2]
```diff
@@ -1,35 +1,40 @@
...
       navigate(TIMELINE.to)
     }
   })
 
-  const calculatePeriodDate = async () => {
+  const onSubmit = async ({ tag, lastPeriod }) => {
     setError(false)
 
+    if (!mainMensesTag) {
+      await dispatch(addMensesTag(tag))
```
Comment (raae) [inline src/pages/incomplete.js:35]: **issue:** No dispatch on the functions from `useSettings`

Comment (SaraVieira) [reply src/pages/incomplete.js:35]: Yeah, already fixed but then Eurovision started 😂

Comment (raae) [reply src/pages/incomplete.js:35]: I approve of your priorities 🤣

Comment (SaraVieira) [reply src/pages/incomplete.js]: Euvosion is done and it is fixed :)

**Comment by SaraVieira** [comment]
> Because of the needs of Netlify functions we are using npm, so remove yarn.lock and use package.lock from main. Otherwise I think Netlify gets confused.

Done and also added yarn.lock to ignore otherwise I'm gonna keep pushing it by accident


also ready for review now :)

File: src/features/profile/Incomplete/IncompleteSettings.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                  <InputAdornment position="start">#</InputAdornment>
+                ),
+              }}
+            />
+          </Box>
+        )}
+        <LastDateInput value={lastPeriod} onChange={setLastPeriod} />
+        <Box mt={2}>
+          <Button type="submit" color="secondary" variant="contained">
+            {!mainMensesTag ? "Save" : "Calculate"}
```
Comment (raae) [inline src/features/profile/Incomplete/IncompleteSettings.js:56]: **nitpick (non-blocking):** I think calculate can work for both cases...

**Review by raae — APPROVED** [review]
🎉🎉🎉:

**Pull request merged**